NoSoft-SA/nspack
# Hotfix/non mixed govt sheets fix

Summary of this change:
Problem: Since https://github.com/NoSoft-SA/nspack/commit/73376ea62501b071ef11a3cd9f5d0a1548bd3fff, non-mixed TM govt inspection sheets did not properly propagate the destination region/country from the sheet header to the individual govt_inspection_pallets. 

Additionally, updating missing masterfiles did not properly reactivate eCert agreements upon updating the eCert agreements.


CHANGELOG entry:

### Fixed
- Refactored the validation of `govt_inspection_pallets` for both mixed/non-mixed sheets and for continuous inspection processes.

**How to test this code via the UI:**
Add a pallet to a govt inspection sheet under the following cases:
- For clients that have a continuous inspection (Kr) and normal inspections (UD,UM,MC,..).
- For mixed and non-mixed TM govt inspection sheets.
Afterwards, check that the destination region/country -ids are properly propagated onto the pallet record.
